### PR TITLE
faq: add blurb about uploading to private AWS regions

### PR DESCRIPTION
**Pull request by jlebon**
This is mostly adapted from the RHCOS FAQ:
https://github.com/openshift/os/blob/master/FAQ.md

But it applies equally well to FCOS:
https://github.com/coreos/fedora-coreos-tracker/issues/383

### CODE REVIEW

File: modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -193,3 +193,20 @@ Kubernetes cluster, you will be able to use upcoming tools to coordinate
 updates and reboots.
 
 https://discussion.fedoraproject.org/t/launch-faq-how-do-i-coordinate-cluster-wide-os-updates-is-locksmith-or-the-container-linux-update-operator-available-for-fedora-coreos/56[discuss]
+
+== How do I upload Fedora CoreOS to private AWS EC2 regions?
+
+Fedora CoreOS today is only uploaded to the public AWS regions. For private
+regions like GovCloud and AWS China, you must upload the images yourself.
```
Comment (bgilbert) [inline modules/ROOT/pages/faq.adoc]: s/public/standard/
`For regions in other AWS partitions like GovCloud and AWS China`...

**Comment by dustymabe** [comment]
This is great! Do you think it would be better if we had a real world example of running those commands with an FCOS release?

File: modules/ROOT/pages/faq.adoc [hunk 2]
```diff
@@ -193,3 +193,20 @@ Kubernetes cluster, you will be able to use upcoming tools to coordinate
...
+
+Note that Fedora CoreOS uses a unified BIOS/UEFI partition layout. As such, it
+is not compatible with the default `aws ec2 import-image` API (for more
```
Comment (bgilbert) [inline modules/ROOT/pages/faq.adoc]: How is it the default?  Can we drop the word `default`?

Comment (jlebon) [reply modules/ROOT/pages/faq.adoc]: It's the "default" insofar as it's what I understand a more common workflow than `import-snapshot` for people who want to create AMIs from local disk images. But not against dropping the word entirely. :)

**Comment by jlebon** [comment]
> This is great! Do you think it would be better if we had a real world example of running those commands with an FCOS release?

Thanks! I think having commands would make sense. Though I wouldn't feel comfortable typing those up without trying them out myself, and I don't have the cycles to try it out now. We can probably leave it as a follow-up?

**Pull request merged**